qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -83,3 +110,38 @@ def _get_capacitance(self) -> Tuple[float, float]:
             raise ValueError("Result format not supported.")
 
         return float(parsed[0]["value"]), float(parsed[1]["value"])
+
+    def _set_phase_compensation_mode(self, mode: constants.ADJ.Mode) -> None:
+        msg = MessageBuilder().adj(chnum=self.channels[0], mode=mode)
+        self.write(msg.message)
+
+    def _query_phase_compensation(self):
+        with self.root_instrument.timeout.set_to(self.time_out_phase_compensation):
+            msg = MessageBuilder().adj_query(chnum=self.channels[0],mode=constants.ADJQuery.Mode.MEASURE)
```

Comment:
please PEP8 the code in this PR :)
```suggestion
            msg = MessageBuilder().adj_query(chnum=self.channels[0], mode=constants.ADJQuery.Mode.MEASURE)
```